### PR TITLE
Interactive code editor 6693

### PR DESCRIPTION
**Pull request by SantiagoDevRel**
Hi guys, I just added the interactive code editor to `getting started`, `Mastering providers`, and `Mastering wallets and accounts`.
First I want to try it out, to see that is working well, track the views/usage, and then implement it in other sections of the docs (also i'll add this after we create the other guides for `utils`, `events`, etc)

#6740

### CODE REVIEW

**Comment by SantiagoDevRel** [comment]
Pls check the `live code editor` section and suggest any improvements to the code sample, or approve if that's fine.

File: docs/docs/guides/getting_started/getting_started.md [hunk 1]
```diff
@@ -246,4 +246,27 @@ const subscription = uniswapToken.events.Transfer();
...
+ts-node index.ts` 
+```
+
+And you'll see the result in the console.
+
+:::info
+To navigate between the different files of the live code project, click on the menu icon "`☰`" located in the top-left corner of the code editor 
+:::
+
+<iframe width="100%" height="600px" frameBorder="0" src="https://stackblitz.com/edit/stackblitz-starters-gfq5gy?embed=1&file=index.js&view=editor"></iframe>
```
Comment (Muhammad-Altabba) [inline docs/docs/guides/getting_started/getting_started.md]:
```suggestion
<iframe width="100%" height="600px" frameBorder="0" src="https://stackblitz.com/edit/stackblitz-starters-gfq5gy?embed=1&file=index.js&view=editor&showSidebar=1"></iframe>  
```

File: docs/docs/guides/wallet/index.md [hunk 2]
```diff
@@ -218,4 +218,28 @@ The following is a list of `Accounts` [methods](/libdocs/Wallet) in the `web3.et
...
+ts-node index.ts` 
+```
+
+And you'll see the result in the console.
+
+:::info
+To navigate between the different files of the live code project, click on the menu icon "`☰`" located in the top-left corner of the code editor 
+:::
+
+<iframe width="100%" height="600px" frameBorder="0" src="https://stackblitz.com/edit/stackblitz-starters-b5suyj?embed=1&file=index.js&view=editor"></iframe>
```
Comment (Muhammad-Altabba) [inline docs/docs/guides/wallet/index.md]:
```suggestion
<iframe width="100%" height="600px" frameBorder="0" src="https://stackblitz.com/edit/stackblitz-starters-b5suyj?embed=1&file=index.js&view=editor&showSidebar=1"></iframe>  
```

File: docs/docs/guides/web3_providers_guide/index.md [hunk 3]
```diff
@@ -372,4 +372,27 @@ provider.on('error', (error) => {
...
+ts-node index.ts` 
+```
+
+And you'll see the result in the console.
+
+:::info
+To navigate between the different files of the live code project, click on the menu icon "`☰`" located in the top-left corner of the code editor 
+:::
+
+<iframe width="100%" height="600px" frameBorder="0" src="https://stackblitz.com/edit/stackblitz-starters-bq5v9y?embed=1&file=index.js&view=editor"></iframe>
```
Comment (Muhammad-Altabba) [inline docs/docs/guides/web3_providers_guide/index.md]:
```suggestion
<iframe width="100%" height="600px" frameBorder="0" src="https://stackblitz.com/edit/stackblitz-starters-bq5v9y?embed=1&file=index.js&view=editor&showSidebar=1"></iframe>  
```

File: docs/docs/guides/web3_providers_guide/index.md [hunk 4]
```diff
@@ -372,4 +372,27 @@ provider.on('error', (error) => {
...
+```
+- For **Typescript** (.ts)
+```bash
+ts-node index.ts` 
+```
+
+And you'll see the result in the console.
+
+:::info
+To navigate between the different files of the live code project, click on the menu icon "`☰`" located in the top-left corner of the code editor
```
Comment (Muhammad-Altabba) [inline docs/docs/guides/web3_providers_guide/index.md]:
```suggestion
To navigate between the different files of the live code project, click on the file name at sidebar on the left.
```

File: docs/docs/guides/wallet/index.md [hunk 5]
```diff
@@ -218,4 +218,28 @@ The following is a list of `Accounts` [methods](/libdocs/Wallet) in the `web3.et
...
+```
+- For **Typescript** (.ts)
+```bash
+ts-node index.ts` 
+```
+
+And you'll see the result in the console.
+
+:::info
+To navigate between the different files of the live code project, click on the menu icon "`☰`" located in the top-left corner of the code editor
```
Comment (Muhammad-Altabba) [inline docs/docs/guides/wallet/index.md]:
```suggestion
To navigate between the different files of the live code project, click on the file name at sidebar on the left.
```

File: docs/docs/guides/getting_started/getting_started.md [hunk 6]
```diff
@@ -246,4 +246,27 @@ const subscription = uniswapToken.events.Transfer();
...
+```
+- For **Typescript** (.ts)
+```bash
+ts-node index.ts` 
+```
+
+And you'll see the result in the console.
+
+:::info
+To navigate between the different files of the live code project, click on the menu icon "`☰`" located in the top-left corner of the code editor
```
Comment (Muhammad-Altabba) [inline docs/docs/guides/getting_started/getting_started.md]:
```suggestion
To navigate between the different files of the live code project, click on the file name at sidebar on the left.
```

**Review by Muhammad-Altabba — COMMENTED** [review]
It seems nice!
I also suggest to show the Side Bar to the left. As provided in the code suggestions.

I got the option from: https://developer.stackblitz.com/guides/integration/embedding#embed-url-options

**Review by avkos — APPROVED** [review]

**Review by jdevcs — COMMENTED** [review]
LGTM except existing comments. also i'll suggest to use web3js acct for https://stackblitz.com/ if possible.

**Review by Muhammad-Altabba — APPROVED** [review]

**Review by jdevcs — APPROVED** [review]

**Pull request merged**